lib/utils.js

```diff
@@ -147,10 +160,22 @@ const utils = {
   },
 
   saveUserConfig: async (config) => {
-    const conf = yaml.safeDump(config);
     const f = resolveUserConfig();
     await ensureFile(f);
-    await writeFile(f, conf);
+
+    let userConfig = yaml.safeLoad(await readFile(f)) || {};
+    if (has('apiToken', userConfig)) {
```

Comment:
why are you doing this re-write here when you are also doing it in `loadUserConfig`? shouldn't it just be in one spot? I'd vote for in loadUserConfig, where it just rewrites the entire object if needed and calls save.